### PR TITLE
[FEATURE] MS-23: Profiles Refactor

### PR DESCRIPTION
**Pull request by JamirMansilla**
**Commits**

- MS-23: rename all the .js files to .jsx
- MS-23: replace old code
- MS-23: replace old code from list component
- MS-23: replace old code from form
- MS-23: add preloader function
- MS-23: delete console.logs and add resource to submit button text
- MS-23: fix preloader issue and delete comments
- MS-23: fix conflicts
- MS-23: fix style issues of form and list

### CODE REVIEW

File: src/Components/Profiles/index.jsx [hunk 1]
```diff
@@ -54,22 +60,24 @@ function Profiles() {
...
+        ) : (
+          <>
+            <List data={profiles} header={tableHeader} openModal={openModal} />
+            <Link to="/profiles/form" className={styles.buttonAdd}>
+              <span className={styles.buttonGreen}>Add Profile</span>
+            </Link>
+          </>
+        )}
       </section>
     </div>
```
Comment (BaltazarCortina) [inline src/Components/Profiles/index.jsx:82]: The div.container shouldn't be there, the modal should be outside of the section and the section should have class "container"

File: src/Components/Profiles/List/list.module.css [hunk 2]
```diff
@@ -29,10 +28,6 @@ th {
   padding: 8px;
 }
 
-.list th:last-child{
-  width: 40%;
-}
-
 .action {
```
Comment (BaltazarCortina) [inline src/Components/Profiles/List/list.module.css:31]: Lines 31 to 39 shouldn't be there

File: src/Components/Profiles/Form/index.jsx [hunk 3]
```diff
@@ -0,0 +1,139 @@
...
+        </button>
+      </form>
+      <Modal
+        showModal={showModal}
+        type={modalType}
+        content={modalContent}
+        closeModalFn={closeModalFn}
+        titleModal={titleModal}
+      />
+    </div>
```
Comment (BaltazarCortina) [inline src/Components/Profiles/Form/index.jsx:135]: Should have the same structure as in candidates

File: src/Components/Profiles/index.jsx (hunk 1 above)
Comment (JamirMansilla) [reply src/Components/Profiles/index.jsx]: solved

**Review by EugenioPonzone — APPROVED** [review]

**Review by BaltazarCortina — APPROVED** [review]

**Pull request merged**